src/wechaty.ts

```diff
@@ -202,6 +215,38 @@ export class Wechaty extends EventEmitter implements Sayable {
     return this.globalInstance
   }
 
+  /**
+   * @param   {WechatyPlugin | WechatyPlugin[]} plugins      - The plugins you want to use
+   *
+   * @return  {Wechaty}                                      - this for chaining,
+   *
+   * @desc
+   * For wechaty ecosystem, allow user to define a 3rd party plugin for the all wechaty instances
+   *
+   * @example
+   * // Report all chat message to my server.
+   *
+   * Class WechatyReportPlugin {
+   *   constructor(options) {
+   *     this.options = options;
+   *   }
+   *   install(bot) {
+   *     bot.on('message', message => http.post(this.options.url, { data: message }))
+   *   }
+   * }
+   *
+   * bot.use(new WechatyReportPlugin({ url: 'http://somewhere.to.report.your.data.com' });
+   */
+  public static use (
+    plugins: WechatyPlugin | WechatyPlugin[],
+  ) {
+    ([] as WechatyPlugin[]).concat(plugins).forEach(plugin => {
```

Comment:
If it is not a must, I feel that we can just set a plugin and forget that.

So the `globalPlugins` might not a must at here?

As I asked the question about this, I'd like to learn more from you about the reason that you want to use a global class method of the `use()`: If it's necessary, we can keep it; if it can be forgotten, then let's just forget it.

Reply:
explained below